### DISCUSSION TITLE
More heat signatures for thermal goggles

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12443] [Ideas and suggestions]
Thermal goggles are very good at seeing warm blooded creatures through walls. Particularly when using a high penetrating turret. So things like fire, flares and overheating electrical systems should create large red areas that could be used to obscure or mask your presence. It would also be interesting if clothes with burn protection such as the hazmat suit reduced your signature on it's protected limbs since it reduces the transfer of heat.

**Comment by mygamingaccount** [comment]
Reactor, engine, smoking supercapacitor.

Maybe the goggles only detect the exact temperatures of the things it was programmed to track. Then it would pick up the edge of the reactor that is exactly at 35.8˚C

**Reply by Hazmat616** [reply]
Yeah using any turret with good penetration is trivial to kill people in a sub. Various things like water, and reactors should in waters case reduce the goggles range and in reactors and stuff you mentioned show up as large red blobs to hide people in their presence

**Comment by vinegar667** [comment]
it has to be balanced additionally tho, e.g. range so its not straight up wallhack

**Reply by Hazmat616** [reply]
The entire purpose of this suggestion is to cause it to be harder to see through walls. Currently you only see people and the suggestion to make water reduce it more than air also is there to limit turret sniping

**Reply by BlackMoons** [reply]
If you wanted to reduce turret sniping, they could replace the heat signature particles with lights/objects. Then you wouldn't see people through walls anymore unless you already poke a hole in the wall.

IMO just having anything (non husk) in the dark sea/subs/etc glow red is still a pretty big advantage, x-ray goggles are kinda OP.

**Comment by G-M-twostay** [comment]
It'd be also be fun to be able to hide(or disguise) yourself on thermal goggles to confuse the enemies. In my mod it's possible to hide from thermal goggles by dropping body temperatures.

**Reply by chunhaqiushif** [reply]
How did you do it, can you show me your mods?
